### ISSUE TITLE
about hyper parameter settings in my own dataset

### ISSUE THREAD

**Issue by momohuangsha** [TreB1eN/InsightFace_Pytorch#182]
hi, @TreB1eN  Thank you for sharing.

I set up a dataset from YouTube video, AFDB, CNBC etc,  with a total of 6587 categories.  Use MTCNN to detect face and align

Training data like this:
![image](https://user-images.githubusercontent.com/48303586/144528306-7ce06451-d8b6-45ad-b5ef-646962a63948.png)
use  MobileFaceNet and ArcFace head 
milestones = [30, 70, 200]   lr=0.01  batch size =64 Train in my notebook(GTX1050ti 4G)

After 70 epochs, the train loss stable at about 3 :
![image](https://user-images.githubusercontent.com/48303586/144529093-22308c65-5495-4bc2-a344-d09589e1d8e5.png)

So is the hyper parameter setting appropriate?
Do you have any good suggestions about hyper parameters,please?

**Comment by KevinRK27** [comment]
How did you make the .idx, .lst, and .bin file?
